internal/pkg/externalplugins/cherrypicker/cherrypicker_test.go

```diff
@@ -833,47 +1039,47 @@ func testCherryPickPRWithLabels(clients localgit.Clients, t *testing.T) {
 						expectedBody := "This is an automated cherry-pick of #2"
 						expectedHead := fmt.Sprintf(botUser.Login+":"+cherryPickBranchFmt, 2, branch)
 						var expectedLabels []string
-						for _, label := range pr(evt).PullRequest.Labels {
+						for _, label := range pr(lb).PullRequest.Labels {
 							expectedLabels = append(expectedLabels, label.Name)
 						}
 						expectedLabels = append(expectedLabels, "type/cherrypick-for-"+branch)
 						var expectedReviewers []string
-						for _, reviewer := range pr(evt).PullRequest.RequestedReviewers {
+						for _, reviewer := range pr(lb).PullRequest.RequestedReviewers {
 							expectedReviewers = append(expectedReviewers, reviewer.Login)
 						}
 						expectedAssignees := []string{"developer"}
 						return fmt.Sprintf(prFmt, expectedTitle, expectedBody, expectedHead,
 							branch, expectedLabels, expectedReviewers, expectedAssignees)
 					}
 
-					expectedPRs := 2
-					if len(tc.prLabels) == 0 {
-						if evt == github.PullRequestActionLabeled {
-							expectedPRs = 0
-						} else {
-							expectedPRs = 1
+					if tc.shouldToggle {
+						expectedPRs := 1
+						if len(ghc.prs) != expectedPRs {
+							t.Errorf("Expected %d PRs, got %d", expectedPRs, len(ghc.prs))
 						}
-					}
-					if len(ghc.prs) != expectedPRs {
-						t.Errorf("Expected %d PRs, got %d", expectedPRs, len(ghc.prs))
-					}
 
-					expectedBranches := []string{"release-1.5", "release-1.6"}
-					seenBranches := make(map[string]bool)
-					for _, p := range ghc.prs {
-						pr := prToString(p)
-						if pr != expectedFn("release-1.5") && pr != expectedFn("release-1.6") {
-							t.Errorf("Unexpected PR:\n%s\nExpected to target one of the following branches: %v", pr, expectedBranches)
+						expectedPrs := make(map[string]string)
```

Comment:
The name of the variable here is the same as above, but the actual corresponding one is the number, and the other is the map, distinct them will be better.

Reply:
Good catch!
Fixed.